### PR TITLE
Add nessus-agent job and release to pipeline and manifest

### PR DESCRIPTION
**Pull request by dlapiduz**
Adds nessus agent to all jobs and sets the key for the agent.

### CODE REVIEW

**Comment by sharms** [comment]
Can we update common_tasks to include hardening and collectd?

**Comment by sharms** [comment]
https://github.com/dlapiduz/cg-deploy-cf/pull/1

**Comment by dlapiduz** [comment]
Merged!

File: pipeline.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ jobs:
       trigger: true
     - get: cf-stemcell
       trigger: false
+    - get: cg-s3-nessus-agent-release
```
Comment (LinuxBozo) [inline pipeline.yml]: @dlapiduz Why the `get` on the release? You don't appear to be doing anything with it? If you want it uploaded during deploy, you need to include in the [`releases` of `cf-deployment-staging` task](https://github.com/18F/cg-deploy-cf/pull/12/files#diff-688ee30f2b0e35b63848f09d2c9bbbb3R50).

Comment (dlapiduz) [inline pipeline.yml]: Ah, makes sense. I knew I was forgetting something

**Pull request merged**